playwright/src/test/java/com/microsoft/playwright/TestBrowserTypeConnect.java

```diff
@@ -253,7 +264,8 @@ void shouldThrowWhenCallingWaitForNavigationAfterDisconnect() throws Interrupted
     }
     assertFalse(browser.isConnected());
     try {
-      page.waitForNavigation(() -> {});
+      page.waitForNavigation(() -> {
```

Comment:
revert?

Reply:
I have formatted using IntelliJ hence the above changes